core/src/main/java/org/openstack4j/api/networking/NetworkingService.java

```diff
@@ -58,7 +59,12 @@
      * @return the LBaaS service
      */
     LoadBalancerService loadbalancers();
-    
+
+    /**
+     * @return the LBaaS V2 service
+     */
+    LbaasV2Service lbaasV2();
```

Comment:
Global comment - I wonder if we should keep the naming consistent - where server/builder/model objects already exist in v1, keep the same name for v2, with the addition of 'v2'.